core/src/main/java/com/klaytn/caver/utils/AccountKeyPublicUtils.java

```diff
@@ -55,4 +65,128 @@ public static AccountKeyPublic decompressKey(String compressedPublicKey) {
                 Numeric.toHexStringWithPrefixZeroPadded(ecPoint.getAffineYCoord().toBigInteger(), 64)
         );
     }
+
+    /**
+     * Creates ECPoint instance using compressed public key
+     * @param compressedPublicKey compressed public key
+     * @return ECPoint
+     */
+    public static ECPoint getECPoint(String compressedPublicKey) {
+        String noPrefixPublicKey = Numeric.cleanHexPrefix(compressedPublicKey);
+        boolean yBit = noPrefixPublicKey.startsWith("03");
+        BigInteger xBN = Numeric.toBigInt(noPrefixPublicKey);
+        X9IntegerConverter x9 = new X9IntegerConverter();
+        byte[] compEnc = x9.integerToBytes(xBN, 1 + x9.getByteLength(CURVE.getCurve()));
+        compEnc[0] = (byte)(yBit ? 0x03 : 0x02);
+        ECPoint ecPoint = CURVE.getCurve().decodePoint(compEnc);
+        return ecPoint;
+    }
+
+    /**
+     * Creates ECPoint instance using ecc public key x,y coordinates
+     * @param x x point
+     * @param y y point
+     * @return ECPoint
+     */
+    public static ECPoint getECPoint(String x, String y) {
+        BigInteger bigIntegerX = Numeric.toBigInt(x);
+        BigInteger bigIntegerY = Numeric.toBigInt(y);
+
+        ECPoint ecPoint = CURVE.getCurve().createPoint(bigIntegerX, bigIntegerY);
+        return ecPoint;
+    }
+
+    /**
+     * Check if the X, Y coordinates included in the public key are valid.
+     * @param compressedPubKey compressed public key
+     * @return valid or not
+     */
+    public static boolean checkPointValid(String compressedPubKey) {
```

Comment:
How about validatePoint or validateXYPoint?

Reply:
validateXYPoint is better. I will modify it.